### ISSUE TITLE
BUG: Doublecmd does not open anymore if fileshare disappears from network

### ISSUE THREAD

**Issue by sascha08-15** [doublecmd/doublecmd#183]
Using Windows 10 with different SMB shares on 2 machines. One machine is offline now so that the tabs (2 if I recall correctly) would be initiated with that offline machine's SMB share. 

Excepted behaviour: Show that the tab cannot be loaded, remain the address in the tab however

Actual behavior: Doublecmd hangs and does not load (loading screen).

OS: Windows 10
DblCmd Version 0.9.10 beta, Rev. 9640M

**Comment by alexx2000** [comment]
Try with DC 1.0, it should be better.

**Comment by gitgt** [comment]
Please write the result or close.

**Comment by oomek** [comment]
It's still an issue for me. DC 1.1.0 freezes for 30-60s on a splash screen when it was closed before with a network drive opened that isn't available anymore.
Same freeze occurs when I select from directory hotlist an entry that points to an unaccessible network drive.

**Comment by majkinetor** [comment]
Yeah, this is still a problem (using v1.0.11). There are still several types of problems, not only dissapearing share, but freeze can happen in several of other situations (such as network inavailability).
